tests/core/test_datasheet.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Silicon Compiler Authors. All Rights Reserved.
+import siliconcompiler
+
+
+def test_datasheet():
+    '''API test for help method
+    '''
+
+    chip = siliconcompiler.Chip('nand2')
+
+    mode = 'default'
+    top = chip.top()
+
+    # inputs
+    chip.set('datasheet', top, 'pin', 'a', 'dir', mode, 'input')
+    chip.set('datasheet', top, 'pin', 'b', 'dir', mode, 'input')
+    chip.set('datasheet', top, 'pin', 'a', 'cap', mode, (0.9, 0.9, 0.9))
+    chip.set('datasheet', top, 'pin', 'b', 'cap', mode, (0.9, 0.9, 0.9))
+
+    # output
+    chip.set('datasheet', top, 'pin', 'z', 'function', mode, '~(a&b)')
+    chip.set('datasheet', top, 'pin', 'z', 'dir', mode, 'output')
+    chip.set('datasheet', top, 'pin', 'z', 'trise', mode, 'a', (0.9, 0.9, 0.9))
+    chip.set('datasheet', top, 'pin', 'z', 'tfall', mode, 'a', (0.9, 0.9, 0.9))
+    chip.set('datasheet', top, 'pin', 'z', 'trise', mode, 'b', (0.9, 0.9, 0.9))
+    chip.set('datasheet', top, 'pin', 'z', 'tfall', mode, 'b', (0.9, 0.9, 0.9))
+    chip.set('datasheet', top, 'pin', 'z', 'tdelayr', mode, 'a', (0.9, 0.9, 0.9))
+    chip.set('datasheet', top, 'pin', 'z', 'tdelayf', mode, 'a', (0.9, 0.9, 0.9))
+    chip.set('datasheet', top, 'pin', 'z', 'tdelayr', mode, 'b', (0.9, 0.9, 0.9))
+    chip.set('datasheet', top, 'pin', 'z', 'tdelayf', mode, 'b', (0.9, 0.9, 0.9))
+
+    # print outputs
+    for key in chip.allkeys():
+        if key[0] == 'datasheet':
+            val = chip.get(*key)
+            if val:
+                print(key, chip.get(*key))
```

Comment:
This test doesn't really do anything, it would fail if you couldn't get the value (but that would be an issue with `allkeys()`). I'm not sure this test adds any value to the CI as is.

Reply:
It's really a usability test to make sure all the feature are there in case we ever decide to remove a feature (b/c we forgot the reason or something). Not a great test, but a place holder. Will improve later...